modell/src/main/kotlin/no/nav/tilbakekreving/behov/FagsysteminfoBehov.kt

```diff
@@ -0,0 +1,10 @@
+package no.nav.tilbakekreving.behov
+
+import no.nav.tilbakekreving.kontrakter.ytelse.Fagsystem
+import no.nav.tilbakekreving.kontrakter.ytelse.Ytelsestype
+
+data class FagsysteminfoBehov(
```

Comment:
Blir dette mer riktig?
```suggestion
data class FagsystemInfoBehov(
```

Reply:
Blir ikke det mer engelsk?

Reply:
jo kanskje det